sympy/physics/qho_1d.py

```diff
@@ -67,12 +76,15 @@ def E_n(n, omega):
 def coherent_state(n, alpha):
     """
     Returns <n|alpha> for the coherent states of 1D harmonic oscillator.
-    See https://en.wikipedia.org/wiki/Coherent_states
+    See https://en.wikipedia.org/wiki/Coherent_states.
```

Comment:
Probably not a good idea to add full stop after a url, people end up copying the url with link and it won't work then.

Reply:
Removed the full stop,